plugins/kubernetes/app/views/kubernetes_roles/_roles_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<script type="text/ng-template" id="kubernetes/kubernetes_roles.tmpl.html">
+  <div>
+    <table class="table table-condensed">
+      <thead>
+      <tr>
+        <th>Role</th>
+        <th>Config File</th>
+        <th>Service Name</th>
+        <th>RAM</th>
+        <th>CPU</th>
+        <th>Replicas</th>
+        <th>Actions</th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr ng-show="!roles.length" ng-hide="roles.length">
+        <td>No project roles have been created yet.</td>
```

Comment:
I forgot to add the colspan.
I just finished some tweaks around this logic, anyway. In summary:
- while we're waiting for the call to come back, we're showing a loading spinner in place of the table
- after the call comes back, if there's no results, we display the table with the headers and a single row with the message above (going to add the colspan)
- if there's any items, we display them